magento/magento2
# [26054-Do not duplicate SEO meta data when duplicating a product]

### Description (*)

we have add the below field will be empty after Product duplication from catalog Product Option
Meta title
Meta keywords
Meta description


### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#26054: Do not duplicate SEO meta data when duplicating a product

### Manual testing scenarios (*)

1. Go to Backed -> Catalog -> Products -> Create a Duplicate From Existing Product & Review 
 You wold get a below field's values is empty
URL key => It is already generated with unique ID
Meta title => Should be empty
Meta keywords => Should be empty
Meta description => Should be empty

### Questions or comments


### Contribution checklist (*)
